### PR TITLE
<Dictionary/>: added `multiple` mode for several items selecting

### PR DESCRIPTION
**Pull request by DRITE**
See #396

### CODE REVIEW

File: src/components/ui/MultipleSelect/MultipleSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                )
+            })
+        },
+        [values]
+    )
+
+    const handleOnChange = React.useCallback(
+        (v: string[]) => {
+            const result: MultivalueSingleValue[] = []
+            v.map(item => result.push({id: item, value: item}))
```
Comment (Dergash) [inline src/components/ui/MultipleSelect/MultipleSelect.tsx:61]: It's either
```tsx
const result = v.map(item => ({ id: item, value: item }))
```
or
```tsx
const result: MultivalueSingleValue[] = []
v.forEach(item => {
    result.push({ id: item, value: item })
})
```

There is no point in mapping if the result is not assigned to something: essentially you've mapped `v` to a new array `[1, 2, 3, ..., v.length - 1]` and then dropped it.

File: src/components/ui/MultipleSelect/MultipleSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    className: string,
+    metaError: string,
+    disabled: boolean,
+    placeholder: string,
+    readOnly: boolean,
+    backgroundColor: string,
+    onDrillDown: () => void,
+}
+
+const MultipleSelect: React.FunctionComponent<MultipleSelectProps> = (props) => {
```
Comment (Dergash) [inline src/components/ui/MultipleSelect/MultipleSelect.tsx:26]: I can't understand the difference with our regular `<Dictionary />` component, implementation looks almost identical aside of `multiple` option passed to the underlying antd `<Select />` and some missing features.

Wouldn't it be easier to make something like this:
```tsx
case FieldType.dictionary:
    resultField = <Dictionary
        {...commonProps}
        multiple={props.widgetFieldMeta.multiple}
        onChange={handleChange}
    />
```
We do support custom options for fields description in widget meta.

Comment (DRITE) [reply src/components/ui/MultipleSelect/MultipleSelect.tsx:26]: @Dergash 
I thought about that. Here some reasons why decided to implement an another component:
- New component has simple implementation.
- As `Dictionary` component can get single value it allows to `drilldown` its value. This behavior is not suitable for `Select` in `multiple` mode.
- Extending `Dictionary`'s implementation leads to increasing complexity of  its testing and overriding, I think.

Comment (DRITE) [reply src/components/ui/MultipleSelect/MultipleSelect.tsx:26]: @Dergash 
I've got it. I'll replace it by refactored `Dictionary`.
But I think its usage will look like:
```
case FieldType.multipleSelect:
    resultField = <Dictionary
        {...commonProps}
        multiple
        onChange={handleChange}
    />
```
Ok?

**Review by Dergash — APPROVED** [review]

**Pull request merged**